internal/policies/apparmor/apparmor.go

```diff
@@ -296,3 +322,124 @@ func removeUnusedAssets(apparmorPath string, filesToKeep []string) (e error) {
 		return os.Remove(parentDir)
 	})
 }
+
+// policiesFromFiles produces a list of policies from a given set of apparmor profiles.
+// A profile can have multiple policies.
+func policiesFromFiles(ctx context.Context, apparmorParserCmd []string, loadedPoliciesFile string, profiles []string, onlyLoaded bool) (policies []string, err error) {
```

Comment:
Oh, it could really be, or many a different context (a small function with fewer parameters?) or it can be than I’m inconsistent too (which I try hard to not be :p) :)